cloud/docker_mock.go

```diff
@@ -100,9 +99,12 @@ func (c *dockerClientMock) GetContainer(context.Context, *host.Host, string) (*t
 	return container, nil
 }
 
-func (c *dockerClientMock) GetDockerLogs(context.Context, *host.Host, string, types.ContainerLogsOptions) (*LogReader, error) {
-	out := bytes.NewBufferString("this is a log message")
-	logs := LogReader{OutReader: out}
+func (c *dockerClientMock) GetDockerLogs(_ context.Context, containerID string, _ *host.Host, _ types.ContainerLogsOptions) (*LogInfo, error) {
+	res := LogInfo{}
+	if containerID == "" { // container not started yet
+		return &res, nil
```

Comment:
Should this return nil? (The function returns a pointer, so by convention if nothing is set in the return value, you might expect a nil pointer.)